google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -1522,6 +1522,7 @@ public UnaryCallable<ReadModifyWriteRow, Row> readModifyWriteRowCallable() {
    *
    * @see ServerStreamingCallable For call styles.
    */
+  @InternalApi
```

Comment:
nit: You can also add a comment, something like:

`@InternalApi("Used in dataflow pipeline")`